.gitignore

```diff
@@ -3,4 +3,5 @@
 *.tokens
 *.interp
 .antlr/
-.env
\ No newline at end of file
+.env
+.idea/
```

Comment:
We recommend that devs add this to their personal .gitignore, rather than the project's.

Reply:
yes right